src/openMVG/localization/CCTagLocalizer.cpp

```diff
@@ -421,10 +422,11 @@ CCTagLocalizer::~CCTagLocalizer()
 // subposes is n-1 as we consider the first camera as the main camera and the 
 // reference frame of the grid
 bool CCTagLocalizer::localizeRig(const std::vector<image::Image<unsigned char> > & vec_imageGrey,
-                              const LocalizerParameters *parameters,
-                              std::vector<cameras::Pinhole_Intrinsic_Radial_K3 > &vec_queryIntrinsics,
-                              const std::vector<geometry::Pose3 > &vec_subPoses,
-                              geometry::Pose3 &rigPose)
+                                 const LocalizerParameters *parameters,
+                                 std::vector<cameras::Pinhole_Intrinsic_Radial_K3 > &vec_queryIntrinsics,
+                                 const std::vector<geometry::Pose3 > &vec_subPoses,
+                                 geometry::Pose3 &rigPose,
```

Comment:
yes, but in a far future the main camera won't be necessarily coincide with the pose of the rig